exampleCourse/elements/clickable-image/clickable-image.mustache

```diff
@@ -0,0 +1,29 @@
+{{#question}}
+<div class="example-course-custom-element">
+    <p class="big example-shared-style">Click on the image this many times: {{number}}</p>
+    <input id="my_hidden_field" name="{{answers_name}}" type="hidden" value="0" />
+    <p><img src="{{image_url}}" width="50px" id="IMark"></p>
```

Comment:
Can we give these components unique ids?  I know this is a demo element but I think it would be good practice to prepend the element uuid to the id fields.